src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/ValidationService.kt

```diff
@@ -62,14 +62,13 @@ class ValidationService(
     val validationMessages = mutableListOf<ValidationMessage>()
 
     if (adjustment.days != null && adjustment.days > 0) {
-      val adjustments = adjustmentService.findByPerson(adjustment.person)
+      val adjustments = adjustmentService.findCurrentAdjustments(adjustment.person, startOfSentenceEnvelope)
       val adaDays = adjustments.filter { it.adjustmentType === AdjustmentType.ADDITIONAL_DAYS_AWARDED }
```

Comment:
The below filter has been moved to the query now so is unnecessary here.. also has changed to >= (below was just >)